metrics/sinks/stackdriver/stackdriver_test.go

```diff
@@ -244,8 +268,52 @@ func TestPreprocessMemoryMetrics(t *testing.T) {
 		},
 	}
 
-	computedMetrics := sink.preprocessMemoryMetrics(metricSet)
+	computedMetrics := sink.computeDerivedMetrics(metricSet)
 
 	as.Equal(int64(96), computedMetrics.MetricValues["memory/bytes_used"].IntValue)
 	as.Equal(int64(13), computedMetrics.MetricValues["memory/minor_page_faults"].IntValue)
 }
+
+func TestTranslateMetricSet(t *testing.T) {
+	as := assert.New(t)
+
+	containerUptime := testTranslateMetric(as, generateIntMetric(1000), "uptime", containerLabels, "kubernetes.io/container/uptime")
```

Comment:
Nit: I think it's easier to read when input and output is near each other

Reply:
done